README.md

```diff
@@ -304,6 +312,52 @@ custom:
         response: './mapping-templates/common-response.vtl' # defaults to {name}.response.vtl
 ```
 
+#### Managing API keys
+
+Since v1.5.0, api keys management is supported. You can pass one or more api keys as an array in the `appSync.apiKeys` property.
+
+The keys can either be a string (name of the key with defaults) or an object of the following shape:
+
+|   property   |      default      | description|
+|--------------| ------------------|------------|
+| name         | *auto-generated*  | Name of the key. This is used under the hood to differentiate keys in the deployment process.<br/><br/>Names are used in the Cfn resource name. Please, keep them short and without spaces or special characters to avoid issues. Key names are case sensitive. |
+| description  | *name of the key*   | A short description for that key |
+| expires      | 1y                | Expiration time for the key. <br/>Can be expressed in seconds or in "human" format. eg: `86400`, `30d`, `2w`, `1y` (See [jkroso/parse-duration](https://github.com/jkroso/parse-duration)).<br/>Min: 1d, max: 1y |
```

Comment:
This description is a little misleading as the implementation will set the key to expire at now() + `expires` on the next deployment.  Would it make sense to support two modes?

* `expiresOn` - absolute expiration expressed by Posix timestamp or ISO 8601 string
* `expiresAfter` - on each deployment, updated to now() + `expiresAfter` 

At a minimum, I would suggest updating this description to explain that the expires date for the key will be bumped by `expires` on each deployment.

Also, it may be prudent to mention that deployments may fail if a key has expired and was removed via Appsync Dynamo TTL with a note mentioning that the key name may be changed to resolve the broken deployment.

Reply:
> Would it make sense to support two modes? 

Yes!! This is something I thought about yesterday too. Wanted to upgrade the PR but I haven't had time yet.

> Also, it may be prudent to mention that deployments may fail if a key has expired and was removed via Appsync Dynamo TTL with a note mentioning that the key name may be changed to resolve the broken deployment.

Sure, Will add that. Thanks